qcodes/utils/installation_info.py

```diff
@@ -0,0 +1,53 @@
+"""
+This module contains helper functions that provide information about how
+QCoDeS is installed and about what other packages are installed along with
+QCoDeS
+"""
+from typing import Dict, List
+import subprocess
+
+
+def _pip_list_parser(pip_list_raw: List[str]) -> Dict[str, Dict[str, str]]:
+    """
+    Parse the raw output of pip list into a dict with package name keys and
+    a dict {'version': ..., 'location': ...} values
+
+    Args:
+        pip_list_raw: a list of strings, one line per list element
+    """
+    out: Dict[str, Dict[str, str]] = {}
+
+    # The first two lines are the header and header-content separator
+    for line in pip_list_raw[2:]:
+        fields = line.split()
+        pkg, ver = fields[0], fields[1]
```

Comment:
This version number cannot be trusted. It reflects the version  at install time for an editable install not necessarily the version at run time. The version number in qcodes.__version__ and qcodes.version.__version__ can however be truested to be updated